.github/workflows/build_docker_images.yml

```diff
@@ -80,3 +81,39 @@ jobs:
           context: etl/socrata_export
           push: true
           tags: atddocker/vz-socrata-export:${{ github.ref == 'refs/heads/production' && 'production' || 'development' }}
+
+      - name: Build and push VZ User & CR3 API image
+        if: steps.changes.outputs.user_cr3_api == 'true'
+        uses: docker/build-push-action@v4
+        with:
+          platforms: linux/amd64,linux/arm64
+          context: api
+          push: true
+          tags: atddocker/vz-user-cr3-api:${{ github.ref == 'refs/heads/production' && 'production' || 'latest' }}
+
+
+      - name: Set environment for ECS update
+        if: steps.changes.outputs.user_cr3_api == 'true'
+        run: |
+          if [ "${{ github.ref }}" == "refs/heads/production" ]; then
+            echo "ECS_CLUSTER=atd-vz-data-cluster-production" >> $GITHUB_ENV
+            echo "ECS_SERVICE=arn:aws:ecs:us-east-1:295525487728:service/atd-vz-data-cluster-production/cr3-and-user-mgmt-api" >> $GITHUB_ENV
+          else
+            echo "ECS_CLUSTER=atd-vz-data-cluster-staging" >> $GITHUB_ENV
+            echo "ECS_SERVICE=arn:aws:ecs:us-east-1:295525487728:service/atd-vz-data-cluster-staging/cr3-and-user-mgmt-api" >> $GITHUB_ENV
+          fi
+
+      - name: Restart ECS service
```

Comment:
it is very cool to see this CI  here!!

two minor things i noticed with the current ECS deployment:
1. It appears we have a test cluster, `vz-api-dev` that can be removed

<img width="451" alt="Screenshot 2024-11-07 at 12 28 13 PM" src="https://github.com/user-attachments/assets/6a83ba3d-779f-4eb2-8f8f-b4022428d0da">

2. The current task that's deployed does not appear to be using the `atddocker/vz-user-cr3-api` image
<img width="881" alt="Screenshot 2024-11-07 at 12 30 17 PM" src="https://github.com/user-attachments/assets/233fb018-99ee-442d-98fc-424607ffde50">

Reply:
Thanks John! I have zapped the dev cluster where I cut my teeth on all this, and I also have updated the task definitions to use the latest and greatest builds on the `atddocker` account.